.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
Just created this file as a convenience for VS Code developers using devcontainers (like myself), but it's by no means necessary